templates/webapi/comments/pagination.html.ep

```diff
@@ -1,54 +1,100 @@
-<nav class="comments-pagination" aria-label="Comments pagination">
-  <ul class="pagination">
-    % if ($comments_pager->current_page == $comments_pager->first_page) {
-        <li class="disabled"><span
-    % }
-    % else {
-        <li><a href="<%= url_with->query([ comments_page => $comments_pager->current_page - 1 ]) %>"
-    % }
-      class="page-link" aria-label="Previous">
-        <span aria-hidden="true">&laquo;</span>
-        <span class="sr-only">Previous</span>
-    % if ($comments_pager->current_page == $comments_pager->first_page) {
-        </span>
-    % }
-    % else {
-        </a>
-    % }
-
-    % my $skipped = 0;
-    % my $skipping_range = 5;
-    % for (my $i = $comments_pager->first_page; $i <= $comments_pager->last_page; $i++) {
-        % # skip rendering page buttons which are too far from the beginning, the end or the current page
-        % unless ($i < $skipping_range || $i + $skipping_range >= $comments_pager->last_page || (abs($comments_pager->current_page - $i) < $skipping_range)) {
-            % # render … between gaps
-            % unless ($skipped) {
-                <li class="disabled"><span class="page-link">…</span></li>
-                % $skipped = 1;
+<div class="row">
+    <div class="col-sm-1"></div>
+    <div class="col-sm-11">
+        <nav class="comments-pagination" aria-label="Comments pagination">
+            <ul class="pagination">
+            % if ($comments_pager->current_page == $comments_pager->first_page) {
+                <li class="disabled"><span
+            % } else {
+                <li><a href="<%= url_with->query([ comments_page => $comments_pager->current_page - 1 ]) %>"
             % }
-            % next;
-        % }
-        % $skipped = 0;
-        %= tag 'li', class => ($i == $comments_pager->current_page ? 'active' : undef) => sub {
-        %    link_to $i => url_with->query([ comments_page => $i ]) => (class => 'page-link')
-        % };
-    % }
-
-    % if ($comments_pager->current_page == $comments_pager->last_page) {
-        <li class="disabled"><span
-    % }
-    % else {
-        <li><a href="<%= url_with->query([ comments_page => $comments_pager->current_page + 1 ]) %>"
-    % }
-        class="page-link" aria-label="Next">
-          <span aria-hidden="true">&raquo;</span>
-          <span class="sr-only">Next</span>
-    % if ($comments_pager->current_page == $comments_pager->last_page) {
-        </span>
-    % }
-    % else {
-        </a>
-    % }
-    </li>
-  </ul>
-</nav>
+                class="page-link" aria-label="Previous">
+                    <span aria-hidden="true">&laquo;</span>
+                    <span class="sr-only">Previous</span>
+            % if ($comments_pager->current_page == $comments_pager->first_page) {
+                </span>
+            % } else {
+                </a>
+            % }
+
+            % my @sizes = (5, 10, 12, 16, 20);
```

Comment:
I consider these magic numbers. At least please explain in comments

Reply:
These numbers are the max slots we could add to the pagination bar for each media size that empirically I have checked that fits for each size.
I add this as code comment

Reply:
There must be a better solution than that. Also see https://github.com/os-autoinst/openQA/pull/3015#discussion_r420127472